web-components/chart/chart.tsx

```diff
@@ -317,6 +326,11 @@ export class CobChart {
     this.setChartWidth();
   }
 
+  componentDidUnload() {
+    // Once the component comes off the page, we clean up the view.
+    this.view.finalize();
```

Comment:
You should put a guard around this in case `componentDidLoad` didn't get far enough to create this.

Just:

```
if (this.view) {
  this.view.finalize();
}
```